airspeed/__init__.py

```diff
@@ -365,7 +365,7 @@ class Text(_Element):
         r'|#[^\{\}a-zA-Z0-9#\*]+|\\.)(.*)$',
         re.S +
         re.I)
-    ESCAPED_CHAR = re.compile(r'\\([\\\$#])')
+    ESCAPED_CHAR = re.compile(r'\\([\$#][^\s]+)')
```

Comment:
What is `\s` inside a `[]` character class? Did you mean just `\S`, ie. a non-whitespace character, without the `[]`?

Reply:
@purcell oops, yes I meant to use `\S`, I've got it fixed now